week4.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>First bullet point "discuss"... to what extent? Do you want them to be able to define/describe/list the factors? As a student, I wouldn't know what to make of "discuss" without some more clarity.</p><p> </p><p>Spelling of "calculations" in second sentence.</p><p> </p><p>What do you mean by "modularize"? As a student, I would find this scary to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>